msteams-platform/TOC.yml

```diff
@@ -5,22 +5,22 @@
       href: overview.md
     - name: What's new?
       href: whats-new.md
-- name: Get Started
+- name: Tutorials and code samples
   items:
   - name: Create an app using C#
     href: tutorials/get-started-dotnet-app-studio.md
   - name: Create an app using Node
     href: tutorials/get-started-nodejs-app-studio.md
   - name: Create an app with the Yeoman generator
     href: tutorials/get-started-yeoman.md
-- name: Samples and templates
+  - name: Extend the Teams client with custom apps
+    href: tutorials/code-samples.md   
+- name: App templates and integrations
   items: 
   - name: Production-ready app templates
     href: samples/app-templates.md
-  - name: Production-ready Shifts Connectors
+  - name: Production-ready Shifts connectors
```

Comment:
I believe "Connectors" here is a product name and should be capitalized, but I'm not 100%